frontend/pages/_app.tsx

```diff
@@ -1,4 +1,6 @@
 import '../styles/globals.css'
+import '../styles/producerProfile.css'
+import 'bootstrap/dist/css/bootstrap.min.css'
```

Comment:
You should move these imports to your component or page because if you import them here they will be applied to every page and component.